### ISSUE TITLE
Uncaught TypeError: Cannot read property 'typeHint' of null

### ISSUE THREAD

**Issue by hultberg** [php-integrator/atom-refactoring#51]
Stumbled upon this error when attempting to generate getters and setts for my class.

I'm using atom-base 3.0.1, and have indexing on save (not continusly)

```
/home/edvin/.dotfiles/atom/.atom/packages/php-integrator-refactoring/lib/GetterSetterProvider.coffee:266

TypeError: Cannot read property 'typeHint' of null
    at GetterSetterProvider.module.exports.GetterSetterProvider.generateSetterForItem (/home/edvin/.atom/packages/php-integrator-refactoring/lib/GetterSetterProvider.coffee:266:49)
    at GetterSetterProvider.module.exports.GetterSetterProvider.onConfirm (/home/edvin/.atom/packages/php-integrator-refactoring/lib/GetterSetterProvider.coffee:205:35)
    at View.module.exports.View.invokeOnDidConfirm (/home/edvin/.dotfiles/atom/.atom/packages/php-integrator-refactoring/lib/GetterSetterProvider/View.coffee:35:13)
    at View.module.exports.MultiSelectionView.confirmedByButton (/home/edvin/.dotfiles/atom/.atom/packages/php-integrator-refactoring/lib/Utility/MultiSelectionView.coffee:208:10)
    at HTMLButtonElement.<anonymous> (/home/edvin/.dotfiles/atom/.atom/packages/php-integrator-refactoring/lib/Utility/MultiSelectionView.coffee:81:14)
    at HTMLDivElement.dispatch (/home/edvin/.dotfiles/atom/.atom/packages/php-integrator-refactoring/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (/home/edvin/.dotfiles/atom/.atom/packages/php-integrator-refactoring/node_modules/jquery/dist/jquery.js:4121:28)
```

**Comment by hultberg** [comment]
Reproduceable steps:
1. Create a class
1. Add two fields, one with `@var` declaration and the other without.
1. Attempt to generate getters and setts for all fields.
1. Observe error.

**Comment by Gert-dev** [comment]
Thanks for reporting this! Coincidentally, I had also noticed this and fixed this in 6b34fb08bab6d2c5c593f0547ca86a4ebcb12d00 a couple of days ago. I've released 1.4.1 that should fix the issue.
